MekHQ/src/mekhq/gui/dialog/CampaignOptionsDialog.java

```diff
@@ -884,6 +887,18 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
                 }
             }
         });
+        
+        useAeroSystemHitsBox.setText(resourceMap.getString("useAeroSystemHits.text")); // NOI18N
```

Comment:
One more small change: can we move this above the "Damage/Destroy Parts by Margin of Failure" checkbox?